src/components/main/friends/request/mod.rs

```diff
@@ -60,22 +60,15 @@ pub fn FriendRequest<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
                 class: "request-controls",
                 div {
                     class: "control-wrap",
-                    if show_skeleton {rsx!(
-                        IconButton {
-                            icon: Shape::XMark,
-                            state: icon_button::State::Secondary,
-                            disabled: true,
-                            on_pressed: move |_| {}
-                        }
-                    )} else {rsx!(
+                    rsx!(
                         IconButton {
                             icon: Shape::XMark,
                             state: icon_button::State::Secondary,
                             on_pressed: move |_| {
                                 cx.props.on_deny.call(());
```

Comment:
This may cause an issue if we don't have a call active and people click it while it's loading.